crates/sui-indexer/src/errors.rs

```diff
@@ -39,9 +39,6 @@ pub enum IndexerError {
 
     #[error("Indexer failed to initialize fullnode RPC client with error: `{0}`")]
     RpcClientInitError(String),
```

Comment:
I would rather not remove this since it's possible that we might change encoding again in the future or somehow the digest for one row is malformed, in which case we don't want to panick the whole indexer because of single bad row